src/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -32,6 +33,27 @@ public class HttpClient : HttpMessageInvoker
         #endregion Fields
 
         #region Properties
+        public static IWebProxy DefaultProxy
+        {
+            get
+            {
+                if (s_defaultProxy == null)
+                {
+                    s_defaultProxy = SystemProxyInfo.Proxy;
+                }
+
+                return s_defaultProxy;
```

Comment:
Nit:
```C#
get => s_defaultProxy ??= SystemProxyInfo.Proxy;
```
?

Reply:
Thx.  I knew about "??".  But I didn't know about the "??=" operator.

Reply:
Actually, scratch that, this should be thread-safe.  It'd be better as something like:
```C#
get => LazyInitializer.EnsureInitialized(ref s_defaultProxy, () => SystemProxyInfo.Proxy);
```

Reply:
Wouldn't it get enough thread safety from the `Lazy<T>` of the SystemProxyInfo.Proxy property?

Reply:
Hmm. Technically / in theory I think a read could be introduced on s_defaultProxy that would cause this to return null on a rare race condition on some hardware, though in practice I don't think that can actually happen.  I was more worried about writes inside of SystemProxyInfo.Proxy getting reordered to after the write to s_defaultProxy, but I see now that you're using `Lazy<T>` in SystemProxyInfo.Proxy, so that won't happen.

Reply:
Ok.  Well I think to be safe I'll use the LazyInitializer.